src/objects/Mesh.js

```diff
@@ -187,7 +187,7 @@ Mesh.prototype = Object.assign( Object.create( Object3D.prototype ), {
 			sphere.copy( geometry.boundingSphere );
 			sphere.applyMatrix4( matrixWorld );
 
-			if ( raycaster.ray.intersectsSphere( sphere ) === false ) return;
+			if ( !sphere.containsPoint( raycaster.ray.origin ) && raycaster.ray.intersectsSphere( sphere ) === false ) return;
```

Comment:
I think it would be better to be consistent and do this:

```js
sphere.containsPoint( raycaster.ray.origin ) === false && raycaster.ray.intersectsSphere( sphere ) === false
```